_posts/2017-04-12-store-libvirt-disk-images-in-home-directory.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Store libvirt disk images in your home directory
```

Comment:
A user who isn't familiar with the problem probably won't understand this title.

Maybe 'How to Make a Pulp Vagrant Env With a Very Large Disk' or similar.